### PR TITLE
Task/update to latest orionplatform

### PR DESCRIPTION
**Pull request by badisa**

### CODE REVIEW

**Comment by badisa** [comment]
There is something up with the changes I made to the manifest.json.

File: {{cookiecutter.project_slug}}/setup.py [hunk 1]
```diff
@@ -23,20 +23,20 @@
 
 # Requirements for {{cookiecutter.module_name}}
 requirements = [
-    "OpenEye-orionplatform==1.0.0",
-    "OpenEye-snowball==0.16.0",
```
Comment (badisa) [inline {{cookiecutter.project_slug}}/setup.py:27]: @mattgeballe This is not available to customers it seems. Can't use orion platform 1.0.0 until Snowball 0.16.0. Should we roll the back to a version of snowball that is publicly available?

Comment (badisa) [reply {{cookiecutter.project_slug}}/setup.py:27]: Found this by testing this on orion.eyesopen.com

Comment (mattgeballe) [reply {{cookiecutter.project_slug}}/setup.py:27]: Probably, at least in the short term until packages go public with the 2019.4 release. looks like snowball v0.14.0 is the most recent public version.

Comment (badisa) [reply {{cookiecutter.project_slug}}/setup.py:27]: Now it defaults to the publicly available orionplatform, but has a comment for switching to snowball.

**Review by mattgeballe — APPROVED** [review]
Looks good!

**Pull request merged**